File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1072.c
#include <stdio.h>

int main(){
    int n, v, vin=0, vout=0, i;
    scanf("%d", &n);

    for(i=0;i<n;i++){
        scanf("%d", &v);
        if(v>=10 && v<=20){
            vin++;
        } else{
            vout++;
        }
    }
    printf("%d in\n", vin);
    printf("%d out\n", vout);

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1072.rb
n = gets.to_i
vin=0
vout=0
    
for i in 1..n do
    v = gets.to_i
    if(v>=10 and v<=20) then
        vin=vin+1;
    else
        vout=vout+1;
    end
end

puts "#{vin} in"
puts "#{vout} out"